testing/kokoro/system_tests.sh

```diff
@@ -131,6 +131,17 @@ if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"system-tests"* ]]; then
   ./testing/kokoro/configure_gcloud.bash;
 fi
 
+# Download and prepare Cloud SQL Proxy
```

Comment:
Please call `testutil.EndToEndTest(t)` in the test cases, and only configure the Cloud SQL Proxy when running E2E tests (see a few lines above).

Might make sense to have a `configure_cloudsql.sh` file to match `configure_gcloud.sh`? Or perhaps rename `configure_gcloud.sh` to `configure_e2e.sh`?

I don't think we need `gcloud` for any non-E2E tests.